daemon/kill.go

```diff
@@ -103,6 +103,13 @@ func (daemon *Daemon) killWithSignal(container *containerpkg.Container, sig int)
 		}
 	}
 
+	if unpause {
```

Comment:
Looks like we try to resume the container, even if the `daemon.kill()` above returned `"container not found"` or `"no such process"`